From 39dc20394ea7e80d932c2e922ed97c19d5b87a1e Mon Sep 17 00:00:00 2001
From: taohaitao <taohaitao@eswincomputing.com>
Date: Wed, 2 Jul 2025 16:45:52 +0800
Subject: [PATCH 593/652] feat:Update vpu7702-pcie-a2.dts.

Changelogs:
Update arch/riscv/boot/dts/eswin/vpu7702-pcie-a2.dts.

Signed-off-by: taohaitao <taohaitao@eswincomputing.com>
---
 arch/riscv/boot/dts/eswin/vpu7702-pcie-a2.dts | 10 +--
 .../boot/dts/eswin/vpu7702_pcie_factory.dts   | 74 +++++++++----------
 2 files changed, 42 insertions(+), 42 deletions(-)

diff --git a/arch/riscv/boot/dts/eswin/vpu7702-pcie-a2.dts b/arch/riscv/boot/dts/eswin/vpu7702-pcie-a2.dts
index 4217d1712555..cdc86bcf6b30 100644
--- a/arch/riscv/boot/dts/eswin/vpu7702-pcie-a2.dts
+++ b/arch/riscv/boot/dts/eswin/vpu7702-pcie-a2.dts
@@ -405,7 +405,7 @@ &gpu0 {
 };
 
 &gc820 {
-	status = "disabled";
+	status = "okay";
 };
 
 &vdec0 {
@@ -827,7 +827,7 @@ &fan_control {
 
 &d0_i2c0 {
 	/* 12V_DC_IN */
-	status = "okay";
+	status = "disabled";
 	i2c-sda-hold-time-ns = <0x40>;
 	ina226@44 {
 		compatible = "ti,ina226";
@@ -861,7 +861,7 @@ npu_vcc1:npu_svcc{
 
 &d0_i2c2 {
 	/* Temperature Sensor */
-	status = "okay";
+	status = "disabled";
 	tmp102@48 {
 		compatible = "ti,tmp102";
 		reg = <0x48>;
@@ -880,7 +880,7 @@ &d0_i2c4 {
 	clock-frequency = <400000>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_i2c10_default>;
-	status = "okay";
+	status = "disabled";
 
 	adc@48 {
 		compatible = "ti,ads1115";
@@ -916,7 +916,7 @@ channel@7 {
 
 &d0_i2c5 {
 	/* AT24C02C EEPROM */
-	status = "okay";
+	status = "disabled";
 	eswin,syscfg = <&d0_sys_con 0x3C0 16>;
 	eeprom@50 {
 		compatible = "atmel,24c02";
diff --git a/arch/riscv/boot/dts/eswin/vpu7702_pcie_factory.dts b/arch/riscv/boot/dts/eswin/vpu7702_pcie_factory.dts
index c31a7615cb6c..5c4cd1f6f851 100644
--- a/arch/riscv/boot/dts/eswin/vpu7702_pcie_factory.dts
+++ b/arch/riscv/boot/dts/eswin/vpu7702_pcie_factory.dts
@@ -826,7 +826,7 @@ &fan_control {
 };
 
 &d0_i2c0 {
-	/* PS_PMBUS */
+	/* 12V_DC_IN */
 	status = "disabled";
 	i2c-sda-hold-time-ns = <0x40>;
 	ina226@44 {
@@ -871,46 +871,11 @@ tmp102@48 {
 };
 
 &d0_i2c3 {
-	/* mipi csi0/csi1 */
+	/* Die 2 Die Serdes */
 	status = "disabled";
 };
 
 &d0_i2c4 {
-	/* mipi csi2/csi3 */
-	status = "disabled";
-};
-
-&d0_i2c5 {
-	/* AT24C02C EEPROM */
-	status = "disabled";
-	eswin,syscfg = <&d0_sys_con 0x3C0 16>;
-	eeprom@50 {
-		compatible = "atmel,24c02";
-		reg = <0x50>;
-	};
-};
-
-&d0_i2c6 {
-	/* unused */
-	status = "disabled";
-};
-
-&d0_i2c7 {
-	/* unused */
-	status = "disabled";
-};
-
-&d0_i2c8 {
-	/* unused */
-	status = "disabled";
-};
-
-&d0_i2c9 {
-	/* unused */
-	status = "disabled";
-};
-
-&d0_aon_i2c0 {
 	/* ADC */
 	clock-frequency = <400000>;
 	pinctrl-names = "default";
@@ -949,6 +914,41 @@ channel@7 {
 	};
 };
 
+&d0_i2c5 {
+	/* AT24C02C EEPROM */
+	status = "disabled";
+	eswin,syscfg = <&d0_sys_con 0x3C0 16>;
+	eeprom@50 {
+		compatible = "atmel,24c02";
+		reg = <0x50>;
+	};
+};
+
+&d0_i2c6 {
+	/* unused */
+	status = "disabled";
+};
+
+&d0_i2c7 {
+	/* unused */
+	status = "disabled";
+};
+
+&d0_i2c8 {
+	/* unused */
+	status = "disabled";
+};
+
+&d0_i2c9 {
+	/* unused */
+	status = "disabled";
+};
+
+&d0_aon_i2c0 {
+	/* SMBUS */
+	status = "disabled";
+};
+
 &d0_aon_i2c1 {
 	status = "disabled";
 };
-- 
2.53.0

